assembl/static2/css/components/assembl-glyph.scss

```diff
@@ -1,11 +1,11 @@
 @font-face {
   font-family: 'assembl-glyph';
-  src: url('#{$fonts-dir}/assembl-glyph.eot?64503037');
```

Comment:
You have a different hash every time you download the font from fontello. It doesn't matter what it is, should be unique that's all.